Examples/Algorithms/TrackFindingExaTrkX/src/TrackFindingAlgorithmExaTrkX.cpp

```diff
@@ -119,6 +119,14 @@ class ExamplesEdmHook : public Acts::ExaTrkXHook {
   }
 };
 
+// TODO do we have these function in the repo somewhere?
+float theta(float r, float z) {
+  return std::atan2(r, z);
+}
+float eta(float r, float z) {
+  return -std::log(std::tan(0.5 * theta(r, z)));
+}
```

Comment:
can this be helpful https://github.com/acts-project/acts/blob/abebe1f42c022f5ecfa0db1b434f8451be25c45e/Core/include/Acts/Utilities/VectorHelpers.hpp#L85-L127

I fear the `-std::log(std::tan(0.5 * theta(r, z)))` is hardcoded a couple of times in the repo. We could clean this out and replace this with a theta input once

Reply:
I try to make this more consistent in a subsequent PR